pkg/controller/storagecluster/quickstarts.go

```diff
@@ -0,0 +1,37 @@
+package storagecluster
+
+import (
+	"context"
+	"github.com/ghodss/yaml"
+	"github.com/go-logr/logr"
+	consolev1 "github.com/openshift/api/console/v1"
+	ocsv1 "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func (r *ReconcileStorageCluster) ensureQuickStarts(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error {
+	for _, qs := range AllQuickStarts {
+		cqs := consolev1.ConsoleQuickStart{}
+		err := yaml.Unmarshal([]byte(qs), &cqs)
```

Comment:
Can we store `Quickstarts` as `[]byte` and make `AllQuickStarts` as `[][]byte`? Gives us a little performance gain.